cockroachcloud/cloud-sso-sql.md

```diff
@@ -0,0 +1,140 @@
+{{site.data.alerts.end}}
+
+**Learn more:**
+
+- [Single Sign-On (SSO) for CockroachDB Cloud organizations](cloud-org-sso.html#cloud-organization-sso)
+- [Configure Cloud Organization Single Sign-On (SSO)](configure-cloud-org-sso.html)
+
+**Prerequisites:**
+
+- You must have a user identity on a {{ site.data.products.db }} organization. For help setting up an organization and cluster, see: [Quickstart with CockroachDB](quickstart.html).
+- SSO must be enabled for your organization. For help configuring SSO for your {{ site.data.products.db }} organization, see: [Configure Cloud Organization Single Sign-On (SSO)](configure-cloud-org-sso.html)
+- SSO must be enabled for your particular {{ site.data.products.db }} user. Configure this at the [{{ site.data.products.db }} Console account settings page](https://cockroachlabs.cloud/account/profile).
+- Your {{ site.data.products.db }} user identity must have access to at least one cluster in your organization.
+- A SQL user specifically corresponding to your SSO identity must be pre-provisioned on the cluster. To authenticate to a specific SQL database, i.e. a cluster, using SSO, a {{ site.data.products.db }} user must have a corresponding SQL user already [created](../{{site.versions["stable"]}}/create-user.html#create-a-user) on that cluster. {{ site.data.products.db }} users must correspond to SQL database users by the convention that the SQL username must be `sso_{email_address}`. 
+
+???!!! {Cameron, how does this format actually work, does the "@gmail" go in there???}
+
+- [`ccloud`, the {{ site.data.products.db }} CLI](ccloud-get-started.html) must be installed on your local workstation.
+
+
+1. First authenticate to your {{ site.data.products.db }} organization. This command will cause your workstation's default browser to open to a {{ site.data.products.db }} authentication portal. Authenticate here as you normally do to the {{ site.data.products.db }} Console. The `ccloud` utility will receive an authentication token from the browser, allowing you to authenticate from the command line.
+
+	{% include_cached copy-clipboard.html %}
+	~~~shell
+	ccloud auth login
+	~~~
+
+1. You may then use the `ccloud` utility to authenticate to your {{ site.data.products.db }} cluster, allowing you to access the SQL interface.
+
+	{% include_cached copy-clipboard.html %}
+	~~~shell
+	ccloud cluster sql --sso {your cluster name}
+	~~~
+
+## Authenticate software users (service accounts) with external IdPs
+
+Currently, {{ site.data.products.db }} can only serve as a token issuer for human users. Authenticating service accounts, i.e. user identities to be controlled by software applications or scripts, rather by humans, is considerably more complicated, as it requires the user, or more realistically, an IdP admin, to provision the appropriate JWT token.
```

Comment:
```suggestion
Currently, {{ site.data.products.db }} can only serve as a token issuer for human users as it requires an interactive flow. Authenticating service accounts, i.e. user identities to be controlled by software applications or scripts, rather than by humans, can not leverage an interactive flow and it needs a different headless mechanism.
```